.eslintrc.json

```diff
@@ -22,10 +25,32 @@
   ],
 
   "rules": {
-    "comma-dangle": ["error", "only-multiline"],
-    "indent": ["error", 2, { "SwitchCase": 1 }],
-    "linebreak-style": ["error", "unix"],
-    "quotes": ["error", "single"],
-    "semi": ["error", "always"]
+    "indent": [
+      2, 
+      4
+    ],
+    "comma-dangle": [
+        "error", 
+        "never"
+    ],
+    "no-unused-vars": [
+        "error", 
+        {
+            "vars": "local", 
+            "args": "none" 
+        }
+    ],
+    "arrow-parens": [
+        2, 
+        "as-needed"
+    ],
+    "no-shadow": "off",
+    "consistent-return": "off",
+    "prefer-promise-reject-errors": "off",
+    "arrow-body-style": "off",
+    "import/no-named-as-default-member": "off",
+    "no-param-reassign": "off",
```

Comment:
Why is this rule turned off? we shouldn't be altering incoming parameters. If we want to alter them we should extract and store them in some other variable. If the parameter was being used somewhere in the function definition already, and you altered it for your own use, it may cause bugs. Thats what this rule will prevent